contributor-growth/drafts/recruiting-playbook.md

```diff
@@ -0,0 +1,133 @@
+ reviewers in a certain area/subject, community managers, project managers, go 
+ developers) and provide any kind of guides, role descriptions, contributor 
+ ladders as you have. The more of these documents that you have available in 
+ your project repository the easier it will be for you in explaining how your 
+ project works to potential new contributors. 
+
+General areas with specifics below:
+- your repo via labels, issues calling out help wanted areas, project boards, 
+and other organization
+- a contributor site/area of the project site for contributors
+- mailing list adverts and promotions (weekly help wanted?)
+- in-person or virtual talks and workshops about the project
+- section during community meeting 
+- project twitter announcements 
+
+## Specific Projects & Programs 
+
+### Your README and project website  
+What: Adding the links to how to contribute and what you are contributing to on 
+your main README is the most effective way 
+Links to good examples: TODO include containerd 'now recruiting'
+
+### Contributor Documentation
+Contributor Guide (for all contributors)
+[TODO include recommendations and links]
+Developer Guide (deep dive for developers)
+[TODO include recommendations and links]
+
+
+### Issue and PR Triage strategy 
+This is the first thing new contributors will see after some kind of interest in
+ your project.
+Good first issues
+Help Wanted
+Be kind.
+In triage and anywhere in your community. if folks are coming around to file 
+issues and submit one-off PRs, being kind can help them want to do it again the 
+next time they think about it. Saying please, thank you, and welcome never 
+hurts. 
+
+### New Contributor Workshops
+[TODO add summary]
+New Contributor Workshops are a live version of your contributing guide with the
+intent of getting a contributor up and running with their first PR, environment
+set up, or some other end goal that leads them to the path quickly. The project
+can explain the norms, address culture and values with a discussion, and make 
+relationships that wouldn't usually happen that fast (or at all). Recording
+the content is a best practice; it can be shared, distributed, and linked at
+any time. This can allievate burden from maintainers on training new 
+contributors. 
+
+#### At targeted events 
+Benefit: form relationships with people in person to create lasting efforts; add
+more TODO   
+Where: KubeCon, Kubernetes Community Days, Cloud Native Events, Project-named 
+events like Contributor Summits  
+Best Practices and past examples: TODO Links
+
+#### Online 
+[TODO - expand]
+Benefit: reach a wider, global audience; add more TODO
+Activites:
+* Governance overview
+* Live PR walk through end to end
+* Code walk through videos
+* Links to talks in conferences
+* Links to presentations/meetups
+Best Practices and past examples: TODO links
+
+## Contributor Playground
+This can be paired with a workshop or on its own for people to try. This gives 
```

Comment:
I don't think most people are familiar with what a contributor playground is. Maybe add a line about

```suggestion
A contributor playground is a non-production repository where contributors can practice submitting pull requests, interacting with any project bots, and safely test out how to participate. This can be paired with a workshop or on its own for people to try. This gives 
```